files/default/sql/ddl/1.1.0__initial_tables.sql

```diff
@@ -1112,6 +1112,51 @@ CREATE TABLE `project_team` (
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- table structure for table `schemas`
+--
+
+/*!40101 set @saved_cs_client     = @@character_set_client */;
+/*!40101 set character_set_client = utf8 */;
+CREATE TABLE `schemas` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `schema` varchar(10000) COLLATE latin1_general_cs NOT NULL,
+  `project_id` int(11) NOT NULL,
```

Comment:
Can you clarify why we need the `project_id` in this table?

Reply:
Avro schemas will now be part of projects rather than global as it is now. `project_id` is necessary because the same schema can be registered from different projects